lighty-core/lighty-common/src/main/java/io/lighty/core/common/models/ModuleId.java

```diff
@@ -72,6 +72,9 @@ public static ModuleId from(final String nameSpace, final String name, final Str
 
     public static ModuleId from(final YangModuleInfo yangModuleInfo) {
         final QName name = yangModuleInfo.getName();
+        if (!name.getRevision().isPresent()) {
```

Comment:
revisions may be missing and that's normal. This class should be updated to work not on strings, but rather on Revisions -- which can be null.